bugshaker/src/main/java/com/github/stkent/bugshaker/utilities/SharedPreferencesUtil.java

```diff
@@ -0,0 +1,51 @@
+package com.github.stkent.bugshaker.utilities;
+
+import java.util.Set;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+
+public class SharedPreferencesUtil {
+
+	private static final String EMAIL_ADDRESS = "emailAddress";
+	private static final String EMAIL_ADDRESS2 = "emailAddress2";
+
+	private static final String KEY_FOR_EMAIL_ADDRESS = "keyForEmailAddress";
+	private static final String KEY_FOR_EMAIL_ADDRESS2 = "keyForEmailAddress2";
```

Comment:
it was in the tutorial that we followed for sharedPreferences and we're not totally sure how to make it work without keys D: